app/src/main/java/org/wikipedia/talk/TalkTopicHolder.kt

```diff
@@ -123,7 +125,10 @@ class TalkTopicHolder internal constructor(
     }
 
     private fun showOverflowMenu(anchorView: View) {
-        context.lifecycleScope.launch {
+        context.lifecycleScope.launch(CoroutineExceptionHandler { _, throwable ->
+            L.e(throwable)
+            FeedbackUtil.showError(context, throwable)
```

Comment:
We should show a snackbar with the error here, but not in the other place, because this spot is directly caused by a user action, whereas the other spot is an indirect query from other flows.